packages/ui/test/council/components/ElectionTabs.test.tsx

```diff
@@ -62,7 +62,7 @@ describe('CouncilTabs', () => {
 
     it('New voting period', async () => {
       window.localStorage.setItem(LocalStorageKey, JSON.stringify('inactive'))
-      stubCouncilAndReferendum(api, 'Election', 'Inactive')
+      stubCouncilAndReferendum(api, 'Announcing', 'Inactive')
```

Comment:
@stepien4soft IMO councilStage: 'Election' and electionStage: 'Inactive' still means "inactive stage" (also I don't think it should happen). If this looks right to you please merge the PR :champagne: